sycl-fusion/passes/CMakeLists.txt

```diff
@@ -44,4 +52,10 @@ target_include_directories(SYCLKernelFusionPasses
   $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
   PRIVATE
   ${SYCL_JIT_BASE_DIR}/common/include
+  ${SYCL_JIT_BASE_DIR}/common/lib
```

Comment:
This shouldn't be included here, see my comment above about the public interface of `sycl-fusion-common`.